src/code/z_construct.c

```diff
@@ -32,8 +32,7 @@ void Interface_Init(PlayState* play) {
 
     parameterSize = (uintptr_t)_parameter_staticSegmentRomEnd - (uintptr_t)_parameter_staticSegmentRomStart;
 
-    // "Permanent PARAMETER Segment = %x"
-    PRINTF("常駐ＰＡＲＡＭＥＴＥＲセグメント=%x\n", parameterSize);
+    PRINTF(T("常駐ＰＡＲＡＭＥＴＥＲセグメント=%x\n", "Permanent PARAMETER Segment = %x\n"), parameterSize);
```

Comment:
it seems to be a bit inconsistent on whether `=` gets spaces around it in this file (the `Textbox game_alloc=%x` doesn't for example). If I had to choose I'd say no spaces to be more faithful to the original strings

Reply:
not a big deal IMO